src/components/search/SearchFilterBar.tsx

```diff
@@ -64,8 +64,11 @@ function ChainSelector({
   const { isOpen, open, close } = useModal();
 
   const multiProvider = useMultiProvider();
-  const chainName = value
-    ? trimToLength(getChainDisplayName(multiProvider, value, true), 12)
+
+  // TODO: chain selector should use domainId
```

Comment:
not immediately clear how to update, will look into it

@jmrossy any pointers here?

Reply:
It already does use domainId doesn't it? See the onCLickChain handler just below

Reply:
clarified - just misleading type